include/version_specifier.h

```diff
@@ -20,7 +20,8 @@
 #define WLCS_VERSION_SPECIFIER_H_
 
 #include <cstdint>
-#include <experimental/optional>
+#include <optional>
+#include <string>
```

Comment:
Unclear why the string include is now required to compile, but it is. Perhaps experimental/optional included it but optional doesn't?

Reply:
Well, we absolutely use `std::string` in the interfaces, so we should definitely be `#include`ing it.